connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -271,17 +256,34 @@ private static Connector instantiateConnector(Class<? extends Connector> connCla
         return result;
     }
 
-    public void stopConnector(String connName) {
+    public void stopConnectors() {
+        stopConnectors(new HashSet<>(connectors.keySet()));
+    }
+
+    public Collection<String> stopConnectors(Collection<String> connectors) {
+        final List<String> stopped = new ArrayList<>(connectors.size());
+        for (String connector: connectors) {
+            if (stopConnector(connector)) {
+                stopped.add(connector);
+            }
+        }
+        return stopped;
```

Comment:
Indeed we don't use the return value, but didn't want to throw away this info as a caller doesn't have any other way of knowing which ones were actually stopped vs ignored. Happy to change to void if you think that's better.

Re: using `stopConnectors()` from revoke. That doesn't sound right to me, wouldn't we only want to stop the set of connectors that has been revoked, rather than all that may be running with this `Worker`?